zoo-css-challenge/style.css

```diff
@@ -1,195 +1,248 @@
 * {
-  font-family: sans-serif;
+    font-family: sans-serif;
 }
 
 header {
-  background-color: #211a1d;
-  color: #ffffff;
-  width: 100%;
-  padding: 20px;
+    background-color: #211a1d;
+    color: #ffffff;
+    width: 100%;
+    padding: 20px;
 }
 
 nav {
-  display: inline-block;
-  width: 100%;
+    display: inline-block;
+    width: 100%;
 }
 
 header ul {
-  margin: 20px 0 0 0;
-  padding: 0;
+    margin: 20px 0 0 0;
+    padding: 0;
 }
 
 header ul li {
-  display: inline-block;
-  margin-right: 10px;
+    display: inline-block;
+    margin-right: 10px;
 }
 
 section {
-  padding: 5%;
-  background-color: #f8f0fb;
+    padding: 5%;
+    background-color: #f8f0fb;
 }
 
+
+/*
 img {
   border: 5px solid #8075ff;
 }
+*/
```

Comment:
Software teams usually don't leave commented code. We can delete the commented code here